plugins/out_loki/loki.c

```diff
@@ -1200,9 +1200,11 @@ static void cb_loki_flush(const void *data, size_t bytes,
     /* User Agent */
     flb_http_add_header(c, "User-Agent", 10, "Fluent-Bit", 10);
 
-    /* Basic Auth headers */
-    if (ctx->http_user && ctx->http_passwd) {
+    /* Auth headers */
+    if (ctx->http_user && ctx->http_passwd) { /* Basic */
         flb_http_basic_auth(c, ctx->http_user, ctx->http_passwd);
+    } else if (ctx->bearer_token) { /* Bearer token */
+        flb_http_bearer_auth(c, ctx->bearer_token);
```

Comment:
Considering that `flb_http_bearer_auth` can fail could you check the result of that function and log errors?